src/main/kotlin/no/nav/syfo/client/journalpostdistribusjon/JournalpostdistribusjonRequest.kt

```diff
@@ -4,4 +4,17 @@ data class JournalpostdistribusjonRequest(
     val journalpostId: String,
     val bestillendeFagsystem: String,
     val dokumentProdApp: String,
+    val distribusjonstype: String = Distribusjonstype.ANNET.name,
```

Comment:
Siden vi bruker distribuerJournalpost bare for fysisk brevutsending til arbeidstakere som har reservert seg mot digital kommuniksjon så spiller ikke disse nye feltene noen rolle - så vi kan bare sette noen defaultverdier.